style3.css

```diff
@@ -93,17 +85,6 @@ ul li:hover ul{
 	visibility: visible;
 }
 
-ul li:hover #a{
```

Comment:
27/09/2021: Because I don't need to create two difference dropdown menu column so I could remove this. Also I have create the div for the logo image above so I could just delete it.